plenum/server/monitor.py

```diff
@@ -286,6 +303,14 @@ def requestOrdered(self, reqIdrs: List[Tuple[str, int]], instId: int,
         might have been reset due to view change due to which this method
         returns None
         """
+        new_messaged = deque()
+        for messaged in self.requestTracker.messaged_reqs:
+            if messaged not in reqIdrs:
+                new_messaged.append(messaged)
+            else:
+                logger.info('Consensus for ReqId: {} was achieved'
+                            .format(messaged[0]))
+        self.requestTracker._messaged_reqs = new_messaged
```

Comment:
Why do we need to re-create the list every time? Why don't just remove the message from `_messaged_reqs` if we ordered it here? I think probability that previously unordered request is finally ordered is not so high, so it will be called relatively rare.